PeerioTechnologies/peerio-mobile
# Fix remove mutations

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/18279/mobile-remove-argument-mutations-when-saving-to-sqlcipher

#### Testing instructions  

Possible regressions: caching module.

1. have being logged in into an account with files, rooms and dms
2. switch to the branch
3. run the app
4. there should be no `error deserializing cached payload`

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
